src/sidebar/services/api.js

```diff
@@ -75,9 +79,6 @@ function stripInternalProperties(obj) {
  * @prop {() => any} onRequestFinished - Callback invoked when the API request finishes.
  */
 
-// istanbul ignore next
-const noop = () => null;
```

Comment:
The need for this was removed by making all fields of the `callbacks` argument to `createAPICall` mandatory.